code/src/pages/MoviesByTheme.js

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react';
+import 'styles/movielist.css';
+import { MovieThumb } from 'components/MovieThumb';
+import { Options } from 'components/Options';
+
+export const MoviesByTheme = () => {
+  const [thememovies, setThemeMovies] = useState([]);
+  const [themeUrl, setThemeUrl] = useState();
+
+    useEffect(() => {
+    fetch(themeUrl)
+    .then(res => res.json())
+    .then(json => {
+      setThemeMovies(json.results);
+    })
+  }, [themeUrl]);
+
+  // Send the results array to MovieThumb component 
+  // where we pick the objects we need
+  return (
+    <>
+      <Options setThemeUrl={setThemeUrl}/>
+      <section className="poster-container">
+        {thememovies.map((movie) => (
+          <MovieThumb key={movie.id} {...movie} />
+        ))}
+      </section>
+    </>
+  );
+};
```

Comment:
I think that you could have applied this functionality to your main fetch. I think maybe we talked about this the other day? That the themeUrl could have a default value of what the first fetch url is, and only reuse it for the other themes :) That way you could remove this entire component. (If you look closely it's pretty much the same code as in the main fetch 🧐😄 )